source/process/casestudy.rst

```diff
@@ -14,8 +14,15 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval (in a shared Google document).
+5. Mattermost case study coordinator:
+
+   -  politely follows up with the customer every 2 business days for an estimate of the draft case study review time
+   -  commissions a banner to be featured in the case study blog post
+   -  creates a draft blog post in WordPress. The draft is then reviewed for formatting, grammar, and other `style guidelines <https://docs.mattermost.com/process/marketing-guidelines.html>`_
+
+6. When the customer has completed a review of the draft case study and approved it, the draft blog post is updated. 
```

Comment:
We should have a step where customer feedback is incorporated into draft blog post in Wordpress and a final draft (using preview mode in Wordpress) is sent to customer as FYI prior to publishing.